content/SCALE/DataProtection/ScrubTasksSCALE.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Scrub Tasks"
+weight: 10
+---
+
+{{< toc >}}
+
+When TrueNAS performs a scrub, ZFS scans the data on a pool.
+Scrubs identify data integrity problems, detect silent data corruptions caused by transient hardware issues, and provide early disk failure alerts.
+
+## Default Scrub Tasks
+
+TrueNAS generates a default scrub task when you create a new pool and sets it to run every Sunday at 12:00 AM.
+
+![ScrubTaskDefaultSCALE](/images/SCALE/ScrubTaskDefaultSCALE.png "Default Scrub Task")
+
+## Creating New Scrub Tasks
+
+TrueNAS needs at least one data [pool]({{< relref "/content/SCALE/Storage/Pools/_index.md" >}}) to create scrub task.
+
+To create a scrub task for a pool, go to **Data Protection** and click **ADD** in the **Scrub Tasks** window.
+
+![ScrubTaskNewSCALE](/images/SCALE/ScrubTaskNewSCALE.png "New Scrub Task")
+
+{{< include file="static/includes/Reference/TasksScrubTasksAddFields.md.part" markdown="true" >}}
+
+{{< expand "Advanced Scheduler" "v" >}}
```

Comment:
I think this advanced scheduler text is a good candidate to turn into a single source snippet and called in multiple places with an include shortcode. The custom scheduler can be opened when configuring Scrub, Cloud Sync, Periodic Snapshot, Rsync, Replication, and SMART tasks (and possibly other locations in the SCALE UI).

Reply:
Fixed!